# Generated by Django 4.2.7 on 2024-01-25 14:50

from django.conf import settings
import django.contrib.auth.models
import django.contrib.auth.validators
from django.db import migrations, models
import django.db.models.deletion
import django.utils.timezone


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        ('account', "0001_initial"),
        ('viewer', '__first__'),
    ]

    operations = [
        migrations.CreateModel(
            name='Profile',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('username', models.CharField(max_length=150, unique=True)),
                ('title', models.CharField(blank=True, max_length=50)),
                ('gender', models.CharField(blank=True, choices=[('M', 'Muž'), ('F', 'Žena'), ('O', 'Ostatní')], max_length=1)),
                ('avatar', models.ImageField(default='avatars/default_avatar.webp', upload_to='avatars/')),
                ('birth_date', models.DateField(blank=True, null=True)),
                ('mobile_number', models.CharField(blank=True, max_length=15)),
                ('address', models.TextField(blank=True)),
                ('op', models.CharField(blank=True, max_length=16)),
                ('borrowed_books', models.ManyToManyField(blank=True, to='viewer.borrowedbook')),
                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='profile', to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
